docs/api-docs/orders/orders-api-overview.md

```diff
@@ -1,780 +1,631 @@
+      "reference_transaction_id": {},
+      "date_created": "2018-05-08T15:06:12+00:00",
+      "avs_result": {...},
+      "cvv_result": {...},
+      "credit_card": {},
+      "gift_certificate": {},
+      "store_credit": {},
+      "offline": {},
+      "custom": {},
+      "payment_instrument_token": {},
+      "payment_method_id": "squarev2.card"
+    }
+  ],
+  "meta": {...}
+}
+```
 
-| Existing Status | Status Passed | Resultant Status | Avalara Tax Document Submission |
-| - | - | - | - |
-| Any | None | `Pending` | None |
-| Paid or `Refunded` | Paid | Paid | None |
-| Unpaid or `Refunded` | Unpaid | Unpaid | None |
-| Paid or `Refunded` | Unpaid | Unpaid | Tax document voided |
-| Unpaid or `Refunded` | Paid | Paid | Tax document submitted |
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
 
-## Overriding preset values
-You can create overrides for calculated values such as product prices, subtotal and totals by sending a fixed value in the request. If values are not supplied for these properties, they will be automatically calculated based on the preset store values and tax rules.
+> ### Note
+> * Depending on the payment method, different information will be available (not all payment gateways return full card or fraud detail).
+> * Transactions are not created for the following payment methods:
+>   * Test Payment Gateway
+>   * PayPal Express
+>   * Amazon Pay
 
-## Calculation of totals
-Automatic calculations for order subtotal and total occur when they are not specified.
+</div>
+</div>
+</div>
 
-You can override order subtotal and/or total. If you choose to override one, we strongly recommend that you override both, because the system will not accurately calculate the other.
+## Handling refunds
 
-Edits to the following properties will trigger a recalculation of the subtotal and total:
+[Orders V3](https://developer.bigcommerce.com/api-reference/store-management/order-transactions) exposes endpoints for managing [order refunds](https://developer.bigcommerce.com/api-reference/store-management/order-transactions/order-refunds). For an overview on using these endpoints, see [Order Refunds in API Docs](https://developer.bigcommerce.com/api-docs/orders/payment-actions).
 
-*   products
-*   discount_amount
-*   shipping_cost_ex_tax
-*   shipping_cost_inc_tax
-*   handling_cost_ex_tax
-*   handling_cost_inc_tax
-*   wrapping_cost_ex_tax
-*   wrapping_cost_inc_tax
-*   billing_address
-*   shipping_addresses
+## Calculating totals
 
-## Order status
-When moving through order management, the order status is not automatically updated. However, you can change this behavior as needed.
+Order `subtotal` and `total` calculate automatically; edits to the following properties trigger recalculation.
 
-You can specify `status_id`, which will automatically set the corresponding status. When `status_id` is not specified, the automatic value is 1 and the status is Pending.
+||
```

Comment:
@aglensmith This should be a list if there's no description or additional information for each term